### PR TITLE
[Translation][Validator] Add missing translations for Japanese (104 - 109)

### LINKED ISSUE #51945: Missing translations for Japanese (ja)
Hello,

There are some missing translation for Japanese (ja). These should be added in branch 5.4. 
See https://github.com/symfony/symfony/issues/38710 for details and [this page](https://symfony-translations.nyholm.tech/#pr) for an example.

These are the files that should be updated: 
- [src/Symfony/Component/Validator/Resources/translations/validators.ja.xlf](https://github.com/symfony/symfony/blob/5.4/src/Symfony/Component/Validator/Resources/translations/validators.ja.xlf)

NOTE: If you want to work on this issue, add a comment to assign it to yourself and let others know that this is already taken.

### PR DESCRIPTION
**Pull request by siketyan**
| Q             | A
| ------------- | ---
| Branch?       | 5.4
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no 
| Tickets       | Fix #51945 
| License       | MIT

This pull request adds a few missing translations of Validator component for Japanese language.

### CODE REVIEW

File: src/Symfony/Component/Validator/Resources/translations/validators.ja.xlf [hunk 1]
```diff
@@ -402,6 +402,30 @@
...
+                <source>The password strength is too low. Please use a stronger password.</source>
+                <target>パスワードの強度が弱すぎます。より強いパスワードを使用してください。</target>
+            </trans-unit>
+            <trans-unit id="106">
+                <source>This value contains characters that are not allowed by the current restriction-level.</source>
+                <target>この値は現在の制限レベルで許可されていない文字を含んでいます。</target>
+            </trans-unit>
+            <trans-unit id="107">
+                <source>Using invisible characters is not allowed.</source>
+                <target>見えない文字は使用できません。</target>
```
Comment (siketyan) [inline src/Symfony/Component/Validator/Resources/translations/validators.ja.xlf]: Other candidates:

```suggestion
                <target>不可視文字は使用できません。</target>
```

Much strict/polite way to say "invisible".

---

```suggestion
                <target>制御文字は使用できません。</target>
```

Although "制御文字" actually means "control characters", it is much common word including invisible characters.

File: src/Symfony/Component/Validator/Resources/translations/validators.ja.xlf [hunk 2]
```diff
@@ -402,6 +402,30 @@
...
+            </trans-unit>
+            <trans-unit id="108">
+                <source>Mixing numbers from different scripts is not allowed.</source>
+                <target>異なる種類の数字を使うことはできません。</target>
```
Comment (siketyan) [inline src/Symfony/Component/Validator/Resources/translations/validators.ja.xlf:423]: Candidates:

```suggestion
                <target>異なる種類の数字を混ぜることはできません。</target>
```

File: src/Symfony/Component/Validator/Resources/translations/validators.ja.xlf [hunk 3]
```diff
@@ -402,6 +402,30 @@
                 <source>The value of the netmask should be between {{ min }} and {{ max }}.</source>
                 <target>ネットマスクの値は、{{ min }}から{{ max }}の間にある必要があります。</target>
             </trans-unit>
+            <trans-unit id="104">
+                <source>The filename is too long. It should have {{ filename_max_length }} character or less.|The filename is too long. It should have {{ filename_max_length }} characters or less.</source>
+                <target>ファイル名が長すぎます。ファイル名の長さは{{ filename_max_length }}文字未満でなければなりません。</target>
```
Comment (mpyw) [inline src/Symfony/Component/Validator/Resources/translations/validators.ja.xlf]: It should be inclusive.

```suggestion
                <target>ファイル名が長すぎます。ファイル名の長さは{{ filename_max_length }}文字以下でなければなりません。</target>
```

File: src/Symfony/Component/Validator/Resources/translations/validators.ja.xlf (hunk 1 above)
Comment (mpyw) [reply src/Symfony/Component/Validator/Resources/translations/validators.ja.xlf]: 👍 for 不可視文字

File: src/Symfony/Component/Validator/Resources/translations/validators.ja.xlf (hunk 3 above)
Comment (siketyan) [reply src/Symfony/Component/Validator/Resources/translations/validators.ja.xlf]: Nice catch, thank you!

File: src/Symfony/Component/Validator/Resources/translations/validators.ja.xlf (hunk 1 above)
Comment (siketyan) [reply src/Symfony/Component/Validator/Resources/translations/validators.ja.xlf]: Changed to "不可視文字" at [52ae2ca](https://github.com/symfony/symfony/pull/51977/commits/52ae2ca78ec3a307f3cc91eb2c69e5f7b38f45f9).

File: src/Symfony/Component/Validator/Resources/translations/validators.ja.xlf [hunk 4]
```diff
@@ -402,6 +402,30 @@
...
+                <source>This value contains characters that are not allowed by the current restriction-level.</source>
+                <target>この値は現在の制限レベルで許可されていない文字を含んでいます。</target>
+            </trans-unit>
+            <trans-unit id="107">
+                <source>Using invisible characters is not allowed.</source>
+                <target>不可視文字は使用できません。</target>
+            </trans-unit>
+            <trans-unit id="108">
+                <source>Mixing numbers from different scripts is not allowed.</source>
+                <target>異なる種類の数字を使うことはできません。</target>
```
Comment (mpyw) [inline src/Symfony/Component/Validator/Resources/translations/validators.ja.xlf:423]: I also prefer this:
 
```suggestion
                <target>異なる文字体系の数字を混在させることはできません。</target>
```

Comment (siketyan) [reply src/Symfony/Component/Validator/Resources/translations/validators.ja.xlf:423]: Hmm, I think "文字体系" is much accurate as the translation of "scripts", but too formal word for this case.

**Comment by 77web** [comment]
memo:
These new messages are used for `NonSuspiciousCharacters` constraint.
Target values will be actually checked by `\Spoofchecker`
https://www.php.net/manual/en/class.spoofchecker.php

**Comment by fabpot** [comment]
Thank you @siketyan.

**Pull request merged**